experimental/packages/opentelemetry-instrumentation-fetch/test/fetch.test.ts

```diff
@@ -321,26 +324,16 @@ describe('fetch', () => {
       api.trace.setSpan(api.context.active(), rootSpan),
       async () => {
         fakeNow = 0;
-        try {
-          const responsePromise = apiCall();
-          fakeNow = 300;
-          const response = await responsePromise;
-
-          // if the url is not ignored, body.read should be called by now
-          // awaiting for the span to end
-          if (readSpy.callCount > 0) await spanEnded;
-
-          // this is a bit tricky as the only way to get all request headers from
-          // fetch is to use json()
-          lastResponse = await response.json();
-          const headers: { [key: string]: string } = {};
-          Object.keys(lastResponse.headers).forEach(key => {
-            headers[key.toLowerCase()] = lastResponse.headers[key];
-          });
-          lastResponse.headers = headers;
```

Comment:
This is the convoluted normalization that didn't turn out to be needed (other than removing this part, most of the diff is just indentation change from removing the now-unecessary try/catch.

As far as I can tell, (other than the previously unfortunate variable naming) all this would do is to normalize the _request_ headers into lower case. However, nothing broke when I stopped doing this, so evidently the two tests that cares don't actually need this.